rust/otap-dataflow/crates/otlp/README.md

```diff
@@ -1,6 +1,91 @@
 # OTLP Nodes
 
-Status: **Skeleton**
+## BatchProcessor
 
-This crate will contain the implementation of the OTLP receiver, OTLP exporter,
-and potentially other OTLP-related processors.
+A generic, async batch processor for the OpenTelemetry Arrow pipeline engine. Buffers incoming data messages and emits them in batches based on batch size, timer ticks, timeout, or shutdown signals.
+
+### Features
+- Buffers messages of any type (`PData`).
+- Emits batches based on batch size, timer ticks, timeout, or shutdown.
+- Handles pipeline control messages (`TimerTick`, `Shutdown`, `Ack`, `Nack`, `Config`).
+- Supports metadata-based batching.
+- Conforms to the `Processor` trait in `otap_df_engine`.
+- Uses `SendEffectHandler` for thread-safe message passing.
```

Comment:
```suggestion
- Uses `EffectHandler` for message passing.
```

Reply:
Except if we have a strong need for a shared/Send processor, I will recommend to start first with a local version of the Processor trait.